.dockerignore

```diff
@@ -1,7 +1,11 @@
+.pytest_cache
+.vagrant
+.vscode
+env
 .env
 .env.template
 .env.test
 .gitignore
-.vagrant/
-env/
+applogs.txt
+poetry.lock
```

Comment:
You shouldn't dockerignore poetry.lock. It's used for controlling specific versions of dependencies so you don't get any mismatch between installations from the same version of the codebase.

Reply:
Removed poetry.lock from .dockerignore